client/cli/src/informant.rs

```diff
@@ -40,6 +41,11 @@ pub fn build(service: &impl AbstractService) -> impl futures::Future<Output = ()
 			} else {
 				trace!(target: "usage", "Usage statistics not displayed as backend does not provide it")
 			}
+			trace!(
```

Comment:
I think this should go to grafana to have nice visualizations.

Reply:
A follow up could be that we integrate some sort of service that sends these memory usage infos to grafana. Each subsystem that can track its own memory usage could report this to this service.

Reply:
(By service I mean something that runs inside of Substrate)

Reply:
Grafana should be done in this pr, IMHO.

There should be some macro to call. @expenses

Reply:
`record_metrics!`?

pretty high level stuff is only written there though

Reply:
Done

Reply:
Ahh okay, I thought that you can call this macro from everywhere, like the telemetry. But seems to be okay for now.